web-animations-2/Overview.bs

```diff
@@ -2113,40 +2124,34 @@ partial dictionary OptionalEffectTiming {
 :   <dfn>delay</dfn>
 ::  Update the description as:
 
-    > The <a>start delay</a> which represents the number of milliseconds from an
-    > <a>animation effect</a>'s <a lt="animation effect start time">start
-    > time</a> to the start of the <a>active interval</a>.
-    > If the <a lt="timeline duration">duration</dfn> of the Animation's
-    > <a>timeline</a> is a percentage, the start delay will be treated as the
-    > result of applying the procedure to convert a
-    > [=time-based animation to a proportional animation=].
+    > The <dfn>specified start delay</dfn> which represents the number of
+    > milliseconds from an <a>animation effect</a>'s
+    > <a lt="animation effect start time">start time</a> to the start of the
+    > <a>active interval</a>.
+    > The |specified start delay| is converted to a <a>start delay</a> following
+    > the [=normalize specified timing=] procedure.
 
 :   <dfn>endDelay</dfn>
 ::  Update the description as:
 
-    > The <a>end delay</a> which represents the number of milliseconds
+    > The <dfn>specified end delay</dfn> which represents the number of milliseconds
     > from the end of an <a>animation effect</a>'s <a>active interval</a>
     > until the <a lt='animation effect start time'>start time</a> of any
-    > <a>animation effect</a> that may
-    > follow, for example, in a <a>sequence effect</a>.
-    > If the <a lt="timeline duration">duration</dfn> of the Animation's
-    > <a>timeline</a> is a percentage, the end delay will be treated as the
-    > result of applying the procedure to convert a
-    > [=time-based animation to a proportional animation=].
-
-</div>
+    > <a>animation effect</a> that may follow, for example, in a <a>sequence
+    >  effect</a>. The |specified end delay| is converted to an <a>end delay</a>
```

Comment:
Nit: Does the `>` in the middle of the `<a>` work?

Reply:
Actually yes. Though I have now reformatted it to avoid any potential issue.